exercises/J-array-get-set/exercise.js

```diff
@@ -4,12 +4,16 @@
   Complete the functions below to get the first and last values from the array
 */
 
-function first(arr) {
-  return; // complete this statement
+function first(numbers) {
+  return numbers[0]; // complete this statement
 }
 
-function last(arr) {
-  return; // complete this statement
+function last(numbers) {
```

Comment:
These functions passes the test, but uses "[magic numbers](https://www.codebyamir.com/blog/software-anti-patterns-magic-numbers)" to determine the last element.

What property of an array can we use to determine the size or length of an array. Knowing the size or length of an array, how do we get the last element ( Hint, an arrays first element is at index zero, so the last is not the same as its length)

Reply:
by using .length -1. Still busy with this one, just need to make it work..

Reply:
Ok, drop me a slack if you stuck anywhere and we can work through it